bin/tachyon-workers.sh

```diff
@@ -15,7 +15,7 @@ if [ $# -le 0 ]; then
 fi
 
 bin=`cd "$( dirname "$0" )"; pwd`
-DEFAULT_LIBEXEC_DIR="$bin"/../libexec
```

Comment:
hahaha, ok I work it out. Also, are you fine with those variable attributions? Eg, *$SOMEDIR/bin/../libexec*